twitchpopups.htm

```diff
@@ -145,6 +197,9 @@ <h1 id="popuptext"></h1>
     // Animate off
     const deleteAnimation = () => {

```

Comment:
should this follow the same format as existing functions?

`function deleteAnimation() {`

Reply:
I just realised that the file includes a mix of formats. :(

Reply:
Ikr!